git_machete/tests/functional/test_machete.py

```diff
@@ -931,6 +931,15 @@ def test_go_up(self) -> None:
             msg="Verify that 'git machete go up' performs 'git checkout' to "
                 "the parent/upstream branch of the current branch."
         )
+        # check short command behaviour
+        self.repo_sandbox.check_out("level-1-branch")
+        self.launch_command("g", "u")
+        self.assertEqual(
+            'level-0-branch',
+            self.launch_command("show", "current").strip(),
+            msg="Verify that 'git machete go up' performs 'git checkout' to "
```

Comment:
```suggestion
            msg="Verify that 'git machete g u' performs 'git checkout' to "
```
Applies to all tests.